stack.yaml

```diff
@@ -1,10 +1,11 @@
-resolver: lts-13.14
+resolver:
+  url: https://raw.githubusercontent.com/commercialhaskell/stackage-snapshots/master/lts/17/5.yaml
+
 packages:
-- .
+  - .
```

Comment:
You can just remove `packages` if you want. `[.]` is the default.

Reply:
Done, thanks!